### DISCUSSION TITLE
What are HSR using for self-localization?

### DISCUSSION THREAD

**Discussion by Shoichi-Hasegawa0628** [HumanSupportRobot/QandA#35] [Q&A] [answered]
What are HSR using for self-localization? Is it MCL？
When launching simulator and input r`ostopic list`, I can see a rosnode (`/pose_integrator`).  
I would like to know from which program this node is started.  

In addition, Is it possible to edit the HSR localization program?

**Comment by hsr-admin** [comment, marked as answer]
Yes, as you may have guessed, HSR self-localization uses MCL!
HSR's localization program is currently not open to the public.
